common/src/main/java/com/microsoft/identity/common/internal/util/DateUtilities.java

```diff
@@ -49,5 +93,17 @@ public static long getExpiresOn(long expiresIn) {
         final long currentTimeSecs = TimeUnit.MILLISECONDS.toSeconds(currentTimeMillis);
         return currentTimeSecs + expiresIn;
     }
+
+    /**
+     * Checks if the provided locale has a default calendar format that is non-Gregorian.
+     *
+     * @param inputLocale A locale to inspect.
+     * @return True, if the provided locale's default calendar format is non-Gregorian. False otherwise.
+     */
+    public static boolean isLocaleCalendarNonGregorian(@NonNull final Locale inputLocale) {
+        final String localeStr = inputLocale.toString();
```

Comment:
thought there was a getLanguage() that we could use instead of this string manipulation.

Reply:
6d16f14855291c43e43a4b0cf3ecab407f359ba3